client/sources/ok_test/sqlite.py

```diff
@@ -105,12 +102,22 @@ def _compare(self, expected, code):
             actual = [e.exception_type]
         else:
             actual = self.format_rows(cursor)
+        self._diff_output(expected, actual)
+
+    def _diff_output(self, expected, actual):
+        expected = expected.split('\n')
+        actual = actual.split('\n')
+
+        if not self.ordered:
+            expected = set(expected)
+            actual = set(actual)
```

Comment:
This will essentially randomize the order the lines are printed below, which could confuse students. Keep the printing order the same but just test if `set(expected) == set(actual)`